index.js

```diff
@@ -69,10 +77,15 @@ async function deleteOlderReleases(keepLatest) {
   try {
     let data = await fetch({
       ...commonOpts,
-      path: `/repos/${owner}/${repo}/releases`,
+      path: `/repos/${owner}/${repo}/releases?per_page=100`,
       method: "GET",
     });
     data = data || [];
+    // filter for delete_pattern
+    data = data.filter(
+      (release) => release.tag_name.indexOf(deletePattern) !== -1
+    );
+
```

Comment:
**Nit:**

May be we can move filter predicate inside next line. Why do extra work if there is compelling reason.

something like:

```suggestion
    // filter for delete_pattern
    const activeMatchedReleases = data.filter(({draft}) => !draft && tag_name.indexOf(deletePattern) !== -1)

```